### PR TITLE
CVM and HTM documentation

### PR DESCRIPTION
**Pull request by mansisandag**
## Proposed changes

Documentation updated for CVM, Inputs and Outputs on wiki

## Impact

Improved documentation

## Types of changes

What types of changes does your code introduce to ABM?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## How has this been tested?

Please describe the tests that you ran to verify your changes.

- [x] Built the documentation locally and confirmed the updates.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: docs/cvm.md [hunk 1]
```diff
@@ -1,7 +1,250 @@
...
+CVM if these trips are made in the context of a commercial vehicle tour pattern. The CVM explicitly 
+distinguishes between residential and non-residential customer types, and between three vehicle types—
+light, medium, and heavy—consistent with the definitions used in the Establishment Survey.
+
+The CVM does not cover the types of work-related travel that would be expected to be covered in the ABM3
+Resident model, namely workers traveling for meetings, sales calls, out-of-town travel, and similar activities.
+The CVM also does not model long-distance freight truck movements that enter and exit the region, which 
+are covered by the Heavy Truck Model (HTM).
+
+The HTM covers long-distance freight movements into and out of San Diego County. The source of the
```
Comment (JiaXu1024) [inline docs/cvm.md:31]: The HTM part could be removed from here.

File: docs/cvm.md [hunk 2]
```diff
@@ -1,7 +1,250 @@
...
+
+• Distinguishing between residential and non-residential customer types;
+
+• Creating linkages between the CVM and the Resident model to better represent household demand 
+for food and package deliveries and for services;
+
+• Leveraging the TNC driver survey to represent this recently emerged type of on-line establishment 
+and the “gig” workers it employs.
+
+In addition, the HTM update to the most current version of FAF5 was important to representing commodity
```
Comment (JiaXu1024) [inline docs/cvm.md:61]: HTM part could be removed (the last paragraph)

File: docs/inputs.md [hunk 3]
```diff
@@ -88,7 +88,11 @@ The table below contains brief descriptions of the input files required to execu
 | [externalExternalTripsByYear.csv](#external_trip) <i> (raw inputs have these by year) <i> | External origin-destination station trip matrix | CSV | Transportation Modelers | |  |  |  | 
 | [externalInternalControlTotalsByYear.csv](#external_internal) <i> (raw inputs have these by year) <i> | External Internal station control totals read by GISDK | CSV | Transportation Modelers | |  |  |  | 
 | [internalExternal_tourTOD.csv](#internal_external_tod) | Internal-External Model tour time-of-day frequency distribution | CSV | Transportation Modelers | 
+| **CVM Establishment Synthesis** |
+| land_use | MGRA based land use file | CSV | |
+| percent_of_establishments_by_luz_size_emp_cat.xlsx | Percent of establishments in LUZ that belong in each size category by industry sector | Excel Workbook | |
 | **Commercial Vehicle Model** (TO BE UPDATED) |  |  |  |
```
Comment (JiaXu1024) [inline docs/inputs.md:94]: remove '(TO BE UPDATED)'

File: docs/inputs.md [hunk 4]
```diff
@@ -88,7 +88,11 @@ The table below contains brief descriptions of the input files required to execu
 | [externalExternalTripsByYear.csv](#external_trip) <i> (raw inputs have these by year) <i> | External origin-destination station trip matrix | CSV | Transportation Modelers | |  |  |  | 
 | [externalInternalControlTotalsByYear.csv](#external_internal) <i> (raw inputs have these by year) <i> | External Internal station control totals read by GISDK | CSV | Transportation Modelers | |  |  |  | 
 | [internalExternal_tourTOD.csv](#internal_external_tod) | Internal-External Model tour time-of-day frequency distribution | CSV | Transportation Modelers | 
+| **CVM Establishment Synthesis** |
+| land_use | MGRA based land use file | CSV | |
+| percent_of_establishments_by_luz_size_emp_cat.xlsx | Percent of establishments in LUZ that belong in each size category by industry sector | Excel Workbook | |
 | **Commercial Vehicle Model** (TO BE UPDATED) |  |  |  | 
+| SynthEstablishments.csv | Output from CVM establishment synthesis, similar description as previous part | CSV | |
 | tazcentroids_cvm.csv | Zone centroid coordinates in state plane feet and albers | CSV | Transportation Modelers |
```
Comment (JiaXu1024) [inline docs/inputs.md:96]: Please check the inputs for Commercial Vehicle Model section, and remove out dated files.

File: docs/inputs.md [hunk 5]
```diff
@@ -1260,6 +1267,167 @@ regional_pass: 10.90
...
+    <tr>
+        <td>Data</td>
+        <td>Detailed industry sector breakdown by size class and LUZ</td>
+    </tr>
+    
+</table>
+
+<a id ="cvm"></a>
+
+### Commercial Vehicle Model ( CVM )
```
Comment (JiaXu1024) [inline docs/inputs.md:1295]: please change line #1295 to "CVM Establishment Synthesis"

File: docs/inputs.md [hunk 6]
```diff
@@ -1260,6 +1267,167 @@ regional_pass: 10.90
...
+#### `SYNTHESTABLISHMENTS.CSV`
```
Comment (JiaXu1024) [inline docs/inputs.md:1296]: Where are the other CVM input files? "MGRAEmpByEstSize.csv", "SummarySynthEstabs.csv"

File: docs/outputs.md [hunk 7]
```diff
@@ -3371,6 +3371,248 @@ This directory contains outputs from the Crossborder model, which represents all
   </tr>
 </table>
 
+### CVM Establishment Synthesis (.\cvm_establishment_synthesis)
```
Comment (JiaXu1024) [inline docs/outputs.md:3374]: This file is an input file for CVM model, please move it to input section

File: docs/outputs.md [hunk 8]
```diff
@@ -3371,6 +3371,248 @@ This directory contains outputs from the Crossborder model, which represents all
...
+   <td>SynthEstablishments.csv
+   </td>
+   <td>Contains information about synthetically generated establishments to be used as inputs to the commercial vehicle model
+   </td>
+  </tr>
+  
+  
+</table>
+
+#### CVM Establishment Synthesis File ( MGRAEmpByEstSize.csv )
```
Comment (JiaXu1024) [inline docs/outputs.md:3406]: This file is an input file for CVM model, please move it to input section

File: docs/outputs.md [hunk 9]
```diff
@@ -3371,6 +3371,248 @@ This directory contains outputs from the Crossborder model, which represents all
...
+    <td>Employment numbers, where Sector is gov, mil, ag_min, bus_svcs, 
+fin_res_mgm, educ, hlth, ret, trn_wrh, con, utl, mnf, whl, ent, accm, food, 
+oth; where Size_Class ranges 1-7.
+
+  </tr>
+  
+
+</table>
+
+#### CVM Establishment Synthesis File ( SummarySynthEstabs.csv )
```
Comment (JiaXu1024) [inline docs/outputs.md:3445]: This file is an input file for CVM model, please move it to input section

File: docs/outputs.md [hunk 10]
```diff
@@ -3371,6 +3371,248 @@ This directory contains outputs from the Crossborder model, which represents all
...
+  </tr>
+  <tr>
+    <td>Count
+    </td>
+    <td>Total number
+  </td>
+  </tr>
+</table>
+
+#### CVM Establishment Synthesis File (SynthEstablishments.csv )
```
Comment (JiaXu1024) [inline docs/outputs.md:3480]: Please remove this file here, it's already in input section

File: docs/inputs.md [hunk 11]
```diff
@@ -88,28 +88,20 @@ The table below contains brief descriptions of the input files required to execu
...
-| FA.csv | Commercial vehicle model parameters file for fleet allocator (FA) industry | CSV | Transportation Modelers | 
-| RE.csv | Commercial vehicle model parameters file for retail (RE) industry | CSV | Transportation Modeler | 
-| SV.csv | Commercial vehicle model parameters file for service (SV) industry | CSV | Transportation Modelers | 
-| TH.csv | Commercial vehicle model parameters file transport and handling (TH) industry | CSV | Transportation Modelers | 
-| WH.csv | Commercial vehicle model parameters file wholesale (WH) industry | CSV | Transportation Modelers | 
+| **CVM Establishment Synthesis** |
+| land_use | MGRA based land use file | CSV | |
+| percent_of_establishments_by_luz_size_emp_cat.xlsx | Percent of establishments in LUZ that belong in each size category by industry sector | Excel Workbook | |
+| **Commercial Vehicle Model** |  |  |  | 
+| SynthEstablishments.csv | Output from CVM establishment synthesis, similar description as previous part | CSV | |
```
Comment (JiaXu1024) [inline docs/inputs.md:95]: Please add the other two inputs here: MGRAEmpByEstSize.csv and SummarySynthEstabs.csv

File: docs/cvm.md (hunk 1 above)
Comment (mansisandag) [reply docs/cvm.md]: Removed HTM

File: docs/cvm.md (hunk 2 above)
Comment (mansisandag) [reply docs/cvm.md]: Done

**Review by JiaXu1024 — APPROVED** [review]
Updates to CVM section look good to me

**Pull request merged**